.github/workflows/documentation.yml

```diff
@@ -0,0 +1,73 @@
+name: Documentation
+
+# Documents the Dart code within this project with the `dartdoc` tool.
+# Publishes the result to a separate branch, `documentation`, under the `docs` folder. 
+# Set GitHub pages to publish that target to get an online documentation site. 
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
I'm no expert in GitHub workflows but this file uses the branch name `master` in multiple places. Should it be changed to `main`?

Reply:
Oh yes, thanks, this is from my old repo before `main` was introduced.